### DISCUSSION TITLE
Generating operations and resolvers on a per-operation basis?

### DISCUSSION THREAD

**Discussion by Atlinx** [twinlogix/typetta#251] [Q&A] [answered]
Is it possible to generate typescript files for each operation (read, write, update, and delete)? I'd like to only use the read operations for my public graphql endpoint, however I'd like to use all the operations for a second private endpoint.

**Comment by edobrb** [comment, marked as answer]
At the moment It's not possible. However you can always use the security policies to negate all write operations.
